mlflow/models/container/__init__.py

```diff
@@ -111,7 +128,7 @@ def _serve_pyfunc(model):
     os.system("python -V")
     os.system('python -c"from mlflow.version import VERSION as V; print(V)"')
     cmd = ("gunicorn --timeout 60 -k gevent -b unix:/tmp/gunicorn.sock -w {nworkers} " +
-           "mlflow.sagemaker.container.scoring_server.wsgi:app").format(nworkers=cpu_count)
+           "mlflow.models.container.scoring_server.wsgi:app").format(nworkers=cpu_count)
```

Comment:
We've reworked some of the sagemaker code to run internally and `cpu_count` from `multiprocessing` doesn't work. More context in this "CPU Issues" section of this post (which is not mine): http://blog.tabanpour.info/projects/2018/09/07/tf-docker-kube.html

Reply:
Thanks @ahutterTA! This might be out of scope for this PR but we will definitely look into that.